apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx

```diff
@@ -6,6 +6,16 @@ export default function useExistingModel({ app, qId, options = {} }) {
   const [modelStore] = useModelStore();
   const { sessionModel } = options;
 
+  const validateOptions = (opts) => {
+    if (opts.dense) {
+      throw new Error('Option "dense" is not avaliable when rendering existing model.');
+    }
+  };
+
+  useEffect(() => {
+    validateOptions(options);
+  }, []);
```

Comment:
Can we just skip the `useEffect`?

Reply:
We should probably iterate through a list of blacklisted options and do the same as you do here. But can wait with that.

Reply:
Yes, there are more to come. But Im thinking its a part of the property-panel work.